aries-site/src/examples/templates/dashboards/components/NotificationItem.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import { Anchor, Box, List, Text } from 'grommet';
+
+export const NotificationItem = ({ defaultItemProps, items, itemProps }) => (
+  <Box>
+    <List
+      data={items}
+      itemProps={itemProps}
+      defaultItemProps={defaultItemProps}
+    >
+      {datum => (
+        <ItemList title={datum.displayName} description={datum.description} />
+      )}
+    </List>
+  </Box>
+);
+
+const ItemList = ({ title, description }) => (
```

Comment:
This naming is also confusing. It indicates a list of things, but it is a single item.

I would also expect these to live in two separate files.